views/prisonerProfile/prisonerCaseNotes/caseNotes.njk

```diff
@@ -72,7 +72,7 @@
             <div id="subTypes">
                 {{ govukSelect({
                     name: 'subType',
-                    id: 'subType',
+                    id: 'sub-type',
```

Comment:
standardise the sub type on the case notes tab with the one on the add case notes page